Sources/NDT7Settings.swift

```diff
@@ -113,110 +80,79 @@ public struct NDT7Timeouts {
     }
 }
 
-/// Mlab NDT7 Server.
-public struct NDT7Server: Codable {
-
-    /// Last server got from MLab
-    static var lastServer: NDT7Server?
-
-    /// ip array
-    public var ip: [String]?
-
-    /// country
-    public var country: String?
-
-    /// city
-    public var city: String?
+public struct LocateAPIV2Response: Codable {
+    public var results: [NDT7ServerV2]
+}
 
-    /// fqdn
-    public var fqdn: String?
+/// Response type for the new, Locate API V2 response
+public struct NDT7ServerV2: Codable {
```

Comment:
please, use NDT7Server with the modifications needed instead to create a new NDT7ServerV2